lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -231,6 +231,26 @@ export class NativeInteractionClient extends BaseInteractionClient {
     protected async handleNativeResponse(response: NativeResponse, request: NativeTokenRequest, reqTimestamp: number): Promise<AuthenticationResult> {
         this.logger.trace("NativeInteractionClient - handleNativeResponse called.");
 
+        // Add Native Broker fields to Telemetry
+        const mats = this.getMATSFromResponse(response);
+        this.performanceClient.addStaticFields({
+            extensionId: this.nativeMessageHandler.getExtensionId(),
+            extensionVersion: this.nativeMessageHandler.getExtensionVersion(),
+            matsBrokerVersion: mats ? mats.broker_version : undefined,
+            matsAccountJoinOnStart: mats ? mats.account_join_on_start : undefined,
+            matsAccountJoinOnEnd: mats ? mats.account_join_on_end : undefined,
+            matsDeviceJoin: mats ? mats.device_join : undefined,
+            matsPromptBehavior: mats ? mats.prompt_behavior : undefined,
```

Comment:
Is all MATS data considered static?

Reply:
Yes, we only get MATS data back once per request